lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -684,6 +660,7 @@ export class PublicClientApplication {
 
         const authCodeRequest: AuthorizationCodeRequest = {
             ...request,
+            redirectUri: request.redirectUri,
```

Comment:
Should this default to the one set in the config?

Reply:
By the time it get's here it has already been set from `request || config || current window`
This is only here because the compiler was complaining it's an optional parameter on the `request` object but required on `AuthorizationCodeRequest`. Not sure if there's a better way to handle this.

Reply:
Ah, makes sense.